pkg/steps/test_test.go

```diff
@@ -0,0 +1,160 @@
+package steps
```

Comment:
`test_test.go`

lol

Reply:
Serious question: should we rename that file to something like `container.go` or `pod.go`? I was considering that.

Reply:
Yeah, you could do that for readability

Reply:
Renamed to `pod.go`